tasks/run-file-updates/run-file-updates.yaml

```diff
@@ -76,26 +86,28 @@ spec:
                            -p ref="${ref}" \
                            -p paths="${updatedPaths}" \
                            -p application=${application} \
+                           -p file_updates_secret="${file_updates_secret}" \
                            -s "$(params.synchronously)" \
-                           -l ${pipelinerun_label}=$(params.pipelineRunUid) \
-                           > $(workspaces.data.path)/ir-result.txt || \
-                           (grep "^\[" $(workspaces.data.path)/ir-result.txt | jq . && exit 1)
-          
-          internalRequest=$(awk 'NR==1{ print $2 }' $(workspaces.data.path)/ir-result.txt | xargs)
-          echo "done (${internalRequest})"
+                           -l "${pipelinerun_label}=$(params.pipelineRunUid)"
 
```

Comment:
Although this version looks better, it considers that the internal-request script will always complete. If it does not complete, the error won't be added to results, as we wanted to do in the start. If the error message in case of timeout or anything is not required in the results anymore, LGTM.

Reply:
currently working to re-use the mocks.sh from rh-sign-image...

Reply:
Got it. Most probably the issues we use to have when the first version was done are not relevant anymore. So this piece is ok for me then.